main.py

```diff
@@ -64,9 +64,8 @@ async def searchTutors(type: str, input: SearchInput, db: Session = Depends(get_
 # create a new user
 @app.post("/createUsers", response_model=UserCreate)
 async def createUsers(user:UserCreate, db: Session = Depends(get_db))-> Registered_User:
-    registered_userID = getUsersByID(user.id, db)
     registered_userEmail = getUsersByEmail(user.email, db)
-    if registered_userID or registered_userEmail:
+    if  registered_userEmail:
```

Comment:
Really good work, I'm going to approve this, but something we have to do later is add some error checking. Not rn, but should be done soon.